src/components/LogInForm/form.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import "./logInForm.css";
+
+export default class Form extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      email: "",
+      nameError: "",
+      emailError: "",
+      send: false,
+      errors: []
+    };
+  }
+
+  handleChange(e) {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+
+    if (this.validate()) {
+      this.setState({ send: true });
+    }
+  }
+
+  validate() {
+    let nameError = "";
+    let emailError = "";
+    let errors = [];
+
+    if (this.state.name.length < 2 && this.state.name.indexOf(" ") === -1) {
+      nameError = "Your login is not correct!";
+      errors.push(nameError);
+    }
+    if (this.state.email.length < 3 && this.state.email.indexOf("@") === -1) {
+      emailError = "Your email is not correct!";
+      errors.push(emailError);
+    }
```

Comment:
This is an extremely naive approach to email validation, but more than this - it's wrong. If the "email" is 3 or more characters long, it doesn't require @ at all. If it contains @, at any position whatsoever, the length is irrelevant. The reason why you missed this fact is due to default behavior of Form and Input tags (Input with type="email" has its own validation) and the form is never even submitted (the submit event never fires at all).